src/test/java/seedu/module/model/ModuleBookTest.java

```diff
@@ -1,63 +1,65 @@
-//                 .build();
-//         List<Module> newModules = Arrays.asList(ALICE, editedAlice);
-//         ModuleBookStub newData = new ModuleBookStub(newModules);
-
-//         assertThrows(DuplicateModuleException.class, () -> addressBook.resetData(newData));
-//     }
-
-//     @Test
-//     public void hasPerson_nullPerson_throwsNullPointerException() {
-//         assertThrows(NullPointerException.class, () -> addressBook.hasModule(null));
-//     }
+package seedu.module.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+//import static org.junit.jupiter.api.Assertions.assertFalse;
+//import static org.junit.jupiter.api.Assertions.assertTrue;
+//import static seedu.module.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+//import static seedu.module.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.module.testutil.Assert.assertThrows;
+//import static seedu.module.testutil.TypicalPersons.ALICE;
+//import static seedu.module.testutil.TypicalPersons.getTypicalAddressBook;
+
+//import java.util.Arrays;
+//import java.util.Collection;
+
+import java.util.Collections;
+//import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+//import javafx.collections.FXCollections;
+//import javafx.collections.ObservableList;
+//import seedu.module.model.module.Module;
+//import seedu.module.model.module.exceptions.DuplicateModuleException;
+//import seedu.module.testutil.PersonBuilder;
+
+public class ModuleBookTest {
+
+    private final ModuleBook addressBook = new ModuleBook();
+
+    @Test
+    public void constructor() {
+        assertEquals(Collections.emptyList(), addressBook.getModuleList());
+    }
+
+    @Test
+    public void resetData_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> addressBook.resetData(null));
+    }
+
+    //     @Test
+    //     public void resetData_withValidReadOnlyAddressBook_replacesData() {
+    //         ModuleBook newData = getTypicalAddressBook();
+    //         addressBook.resetData(newData);
+    //         assertEquals(newData, addressBook);
+    //     }
+
+    //     @Test
+    //     public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
+    //         // Two persons with the same identity fields
+    //         Module editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
+    //                 .build();
+    //         List<Module> newModules = Arrays.asList(ALICE, editedAlice);
+    //         ModuleBookStub newData = new ModuleBookStub(newModules);
+
+    //         assertThrows(DuplicateModuleException.class, () -> addressBook.resetData(newData));
+    //      }
+
+    @Test
+    public void hasPerson_nullPerson_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> addressBook.hasModule(null));
+    }
+}
```

Comment:
In my opinion, you shouldn't add tests for unrelated portions of the code. This PR is about the deadline feature, hence it should have tests related to deadline. For tests on other portions of the code, it should probably go into a new PR, for better management.

Reply:
ok, noted!